mmdet/models/anchor_heads/fovea_head.py

```diff
@@ -2,8 +2,11 @@
 import torch.nn as nn
 from mmcv.cnn import normal_init
 
+import mmdet
 from mmdet.core import multi_apply, multiclass_nms
-from mmdet.ops import DeformConv
+if not mmdet.version.CPU_ONLY:
+    from mmdet.ops import DeformConv
+
```

Comment:
We could try to move these kinds of check inside ops like here https://github.com/open-mmlab/mmdetection/blob/dd88ed65131da79bf60bfaf847236ad7c41a6dad/mmdet/ops/dcn/deform_conv.py#L10

Reply:
i followed your suggestion and did that, please look at my latest commit